cw_ethereum/lib/ethereum_wallet.dart

```diff
@@ -508,4 +508,6 @@ abstract class EthereumWalletBase
   @override
   String signMessage(String message, {String? address = null}) =>
       bytesToHex(_ethPrivateKey.signPersonalMessageToUint8List(ascii.encode(message)));
+
+  dynamic getWeb3Client() => _client.getWeb3Client();
```

Comment:
same
also after modifying it, please follow the flow for any modification needed (ex. casting, etc..)